chain/client/src/test_utils.rs

```diff
@@ -1493,6 +1495,16 @@ impl TestEnv {
         }
     }
 
+    #[track_caller]
```

Comment:
What is this magic :)

Reply:
A thing you stick on top of test helpers. 

If you do 

```
fn foo() {
    bar()
}

#[track_caller]
fn bar() {
    panic!("kaboom")
}
```

then the panic message would point at the `bar()` call inside `foo`, rather than at the `panic` inside of `bar()`.


In this case, this helper can panic if the transaction is not available, and its useful too see the `#[test]` where it happened, rather than the helper iteslf.